local/localstack/start-jupyter.sh

```
@@ -0,0 +1,19 @@
+export PYTHONPATH="/home/hadoop/workspace:${PYTHONPATH}"
+
+# Create Jupyter workspace directory if it doesn't exist
+mkdir -p /home/hadoop/workspace/jupyter_workspace
+
+# Start Jupyter Lab
+cd /home/hadoop/workspace
+jupyter lab --ip=0.0.0.0 --port=8888 --no-browser --allow-root --NotebookApp.token='' --NotebookApp.password='' --notebook-dir=/home/hadoop/workspace/jupyter_workspace
+
+# If JupyterLab exits, keep container running unless specifically terminated
+if [ $? -ne 0 ]; then
+    echo "JupyterLab exited with error code $?"
```

Shellcheck (0.10.0)

[warning] This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten.

(SC2319)